src/Trixi.jl

```diff
@@ -24,7 +24,7 @@ using Printf: @printf, @sprintf, println
 using Reexport: @reexport
 
 import DiffEqBase: CallbackSet, DiscreteCallback,
-                   ODEProblem, ODESolution, ODEFunction,
+                   SplitODEProblem, ODEProblem, ODESolution, ODEFunction,
```

Comment:
Why do we need `SplitODEProblem`?

Reply:
This is from my chemical networks implementation, I just forgot to clean it up.

Reply:
done in  869d241